scripts/rundeck-failover.sh

```
@@ -0,0 +1,518 @@
+    log_info "Login Debug: Executing login curl command..."
+    curl -s -c "$cookie_file" \
+        -d "j_username=${RUNDECK_USER}&j_password=${RUNDECK_PASS}" \
+        "http://localhost:${port}/j_security_check"
+
+    log_info "Login Debug: Checking cookie file after login..."
+    [ -f "$cookie_file" ] && log_info "Cookie file exists" || log_error "Cookie file missing"
+
+    # 1回目のAPIテスト
+    log_info "Login Debug: First API test within login function..."
+    log_info "Login Debug: Executing: curl -s -b $cookie_file http://localhost:$port/api/14/system/info"
+    local test_response1=$(curl -s -b "$cookie_file" "http://localhost:${port}/api/14/system/info")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)